### ISSUE TITLE
on:mouseenter on:mouseleave not working

### ISSUE THREAD

**Issue by 0xmycf** [plentico/plenti#201]
Hello!
I was playing around with plenti and wanted to implement a hover-able element using the on:mouseenter and on:mouseleave events. Similar to this: https://svelte.dev/repl/41a12ececb8744d7bc937933ad2ffa49?version=3.22.3

As it turns out this doesn't work with plenti, but I am not sure if this is intended/expected behavior or not.

I tested it against a pure  svelte built and the events work like expected there.

Any feedback would be greatly appreciated. Thank you!

(This is the component I tried to use:)

```svelte
<script>
    export let text, translation;
    
    let show = false;

    let onHover = () => {
        show = !show;
        console.log(translation);
      }
</script>

<div
  on:mouseenter={onHover}
  on:mouseleave={onHover}>
    { text }
</div>

{#if show}
  <p>
    Show is set to {show}
  </p>
{/if}
```

**Comment by jimafisk** [comment]
Hi @0xmycf, thanks for the bug report! Do you have a repo I could take a look at?

**Comment by 0xmycf** [comment]
Yes, this is the plenti repo I've been using: https://github.com/0xmycf/chinese-learning-website .
This is the project I've tested the events on: https://github.com/0xmycf/svelte-test

I didint configure anything in the second one, just switched the src/App.svelte file.

**Comment by jimafisk** [comment]
It looks like the issue was that the logic in `layouts/scripts/stores.svelte` was removed, but the `layouts/components/incrementer.svelte` and `layouts/component/decrementer.svelte` components were still trying to use it. I just removed these entirely since they didn't seem to be used in your app. That cleared up some errors that were preventing the SPA from hydrating properly, and thus blocking other JS that was trying to run. 

Here's a PR for those changes: https://github.com/0xmycf/chinese-learning-website/pull/1

Seems to work on my machine now:

![plenti](https://user-images.githubusercontent.com/5913244/176307922-c644aaff-dd42-4426-bdc3-d0447f5e5ebe.gif)

**Comment by 0xmycf** [comment]
Ah, interesting... I didn't see the connection between these parts... Thank you!
